tests/parm/nems.configure.hafs_atm_wav.IN

```diff
@@ -41,36 +44,10 @@ ATM_attributes::
   write_restart_at_endofrun = .true.
 ::
 
-# OCN #
```

Comment:
What happened to these settings?

Reply:
This test does not use OCN component.

Reply:
Ah---good catch!

Reply:
Now that I look more carefully, two arrays at the end of this file `ocn_export_fields` and `ocn_import_fields` are probably not needed, but I'm not sure.
@BinLiu-NOAA